elk/evaluate/evaluate.py

```diff
@@ -0,0 +1,58 @@
+import csv
+
+import torch
+
+from elk.training.preprocessing import load_hidden_states, normalize
+
+
+def evaluate(args):
+    """
+    Note: eval is a reserved keyword in python, therefore we use evaluate instead
```

Comment:
You can actually shadow the built in keyword if you want, it won't raise a SyntaxError

Reply:
(Not saying this means you _should_ name it `eval`)